components/lists.css

```diff
@@ -0,0 +1,40 @@
+@import url(/stylesheets/main.css);
+
+.list-page-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.list-container{
+    width: 20rem;
+    height: 15rem;
+    background-color: hsl(var(--primary-lighter));
```

Comment:
you can add hsl color code directly in your variable declaration page, just a suggestion.

Reply:
will keep that in mind